### PR TITLE
LPS-121998 Add new service Impl methods to account for the new Country fields

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke @pei-jung

https://issues.liferay.com/browse/LPS-121998

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by alee8888** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java [hunk 1]
```diff
@@ -14,11 +14,254 @@
...
  * @see CountryLocalServiceBaseImpl
  */
 public class CountryLocalServiceImpl extends CountryLocalServiceBaseImpl {
+
+	@Override
+	public Country addCountry(
+			String a2, String a3, boolean active, boolean billingAllowed,
+			String idd, String name, String number, double position,
+			boolean shippingAllowed, boolean subjectToVAT,
+			Map<String, String> titleMap, ServiceContext serviceContext)
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java:50]: We need to include `boolean zipRequired` here as well.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by alee8888** [comment]
ci:stop

**Comment by alee8888** [comment]
Resent here https://github.com/drewbrokke/liferay-portal/pull/1238

**Pull request closed without merging**